deeprvat/seed_gene_discovery/seed_gene_base_configurations.yaml

```diff
@@ -0,0 +1,16 @@
+
+n_chunks: 15
+
+#debug: True
+
+data:
+    dataset_config:
+        use_common_variants: False
+        use_rare_variants: True
+        rare_embedding:
+            config:
+                verbose: True
+        verbose: True
+    dataloader_config:
+        num_workers: 10
+        #batch_size: 20
```

Comment:
remove this commented out batch size line entirely. It must not be set since the pipeline will fail otherwise. We addressed this in #108.

Reply:
Gotcha, synced up. Fixed with [d5843ef](https://github.com/PMBio/deeprvat/pull/99/commits/d5843ef198c033e289778f8f1a7377a6e9bb9703)